### PR TITLE
Integrate the ngeo mobile app in c2cgeoportal

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: c2cgeoportal/scaffolds/create/+package+/templates/mobile.html_tmpl [hunk 1]
```diff
@@ -36,8 +105,21 @@
     <script src="${request.route_url('deps.js')}"></script>
     <script src="${request.static_url('{{package}}:static-ngeo/js/mobile.js')}"></script>
     <script src="${request.static_url('{{package}}:static-ngeo/build/templatecache.js')}"></script>
+    <script src="${request.static_url('%s/utils/watchwatchers.js' % closure_library_path)}"></script>
+    <script src="${request.static_url('%s/less/dist/less.min.js' % node_modules_path)}"></script>
 % else:
-    <script src="${request.static_url('{{package}}:static-ngeo/build/mobile.js')}"></script>
+    <script src="${request.static_url('demo:static-ngeo/js/mobile.js')}"></script>
```
Comment (ger-benjamin) [inline c2cgeoportal/scaffolds/create/+package+/templates/mobile.html_tmpl]: Is 'demo' correct ?'

File: doc/developer/ngeo.rst [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+   +    <script src="${request.static_url('%s/d3/d3.min.js' % node_modules_path)}"></script>
+   +    <script src="${request.static_url('%s/typeahead.js/dist/typeahead.bundle.js' % node_modules_path)}"></script>
+   +    <script src="${request.static_url('%s/closure/goog/base.js' % closure_library_path)}"></script>
+   +    <script src="${request.route_url('deps.js')}"></script>
+   +    <script src="${request.static_url('{{package}}:static-ngeo/js/mobile.js')}"></script>
+   +    <script src="${request.static_url('{{package}}:static-ngeo/build/templatecache.js')}"></script>
+   +    <script src="${request.static_url('%s/utils/watchwatchers.js' % closure_library_path)}"></script>
+   +    <script src="${request.static_url('%s/less/dist/less.min.js' % node_modules_path)}"></script>
+   +% else:
+   +    <script src="${request.static_url('demo:static-ngeo/js/mobile.js')}"></script>
```
Comment (ger-benjamin) [inline doc/developer/ngeo.rst]: Is 'demo' correct ?

Comment (sbrunner) [inline doc/developer/ngeo.rst]: no, thanks :-)

**Comment by ger-benjamin** [comment]
Not tested but seems correct.

**Pull request merged**